src/RobotSimulator/ArenaConfigBuilder.ts

```diff
@@ -0,0 +1,239 @@
+import { WorldConfig } from "@fruk/simulator-core/dist/engine/SimulatorConfig";
+import {
+  IBallSpec,
+  IBoxSpec,
+  IConeSpec,
+} from "@fruk/simulator-core/dist/engine/specs/CoreSpecs";
+
+export interface ArenaConfig {
+  worldConfig: WorldConfig;
+  ballSpecs?: IBallSpec[];
+  boxSpecs?: IBoxSpec[];
+  coneSpecs?: IConeSpec[];
+}
+
+export function setupPlainArena(): ArenaConfig {
+  const arenaConfig: ArenaConfig = {
```

Comment:
At the moment, adding/removing/modifying an arena would involve editing this file as well as `VMControls.tsx`. We can probably consolidate the configs by storing all of the information in one place, and maybe even divorce the arena configurations from the code entirely. 

How about something like:
- each arena config is stored in an individual JSON file under a directory `ArenaConfigs`
- an `ArenaLoader` class looks in this directory for all JSON files, `JSON.parse(filename)`'s each config, and returns this list of arenas when prompted (e.g. `ArenaLoader.loadArenasFromDir()`)
- each arena configuration JSON could hold all the necessary details about the arena, including the arena's display name e.g. "Parking Lot". Then `VMControls.tsx` wouldn't need to know anything about the arenas as it could just be passed the array of arenas, and adding a new arena would be as simple as adding a new file to the `ArenaConfigs` directory.

One of advantage of this is that the arenas could be deployed separately from the code, which might be useful in the future (e.g. if an arena needs to be kept secret until the start of a competition and so shouldn't be packaged with the code).